Podcast/ProfileHeaderView.swift

```diff
@@ -0,0 +1,254 @@
+//
+//  ProfileHeaderView.swift
+//  Podcast
+//
+//  Created by Drew Dunne on 10/17/16.
+//  Copyright © 2016 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+struct PHVConstants {
+    static let height:CGFloat = 286+138+37
+    static let miniHeight:CGFloat = 85
+    static let bottomBarDist:CGFloat = 226
+    static let statusBarHeight:CGFloat = 20
+}
+
+protocol ProfileHeaderViewDelegate {
+    func followButtonPressed(follow: Bool)
+    func buttonBarPressed(buttonBarNum: Int)
+    func collectionViewCellDidSelectItemAtPath(path: IndexPath)
+}
+
+class ProfileHeaderView: UIView, UICollectionViewDelegate {
+    
+    let fullHeight:CGFloat = PHVConstants.height
+    
+    let statusBarHeight:CGFloat = PHVConstants.statusBarHeight
+    let bottomBarDist:CGFloat = PHVConstants.bottomBarDist
+    let bottomBarHeight:CGFloat = 60
+    let subscriptionsHeight: CGFloat = 138
+    
+    var profileArea: UIView!
+    var usernameLabel: UILabel!
+    var nameLabel: UILabel!
+    var profileImage: UIImageView!
+    var followButton: FollowButton!
+    
+    var subsHeader: ProfileSectionHeaderView!
+    var subscriptions: UICollectionView!
+    
+    // Currently: Followers, Following (Used to include subscriptions)
+    var bottomButton1: UIButton!
+    var bottomButton2: UIButton!
```

Comment:
Right, I only did this because the design team hadn't decided on exactly what the buttons would be. They kept changing it so rather than renaming over and over again I just did this. But I can put it back now.